### PR TITLE
New copyright headers will be copyrighted to Flyte Authors.

### PR DESCRIPTION
**Pull request by kumare3**
Signed-off-by: Ketan Umare <ketan.umare@gmail.com>

# TL;DR
 - Standardized contributing guidelines
 - Readme
 - Also added additional copyright to Flyte Authors.

## Type
 - [x] Docs only

### CODE REVIEW

**Comment by kanterov** [comment]
@kumare3 can you do `mvn spotless:apply`?

**Comment by kumare3** [comment]
> @kumare3 can you do `mvn spotless:apply`?

Will do

**Review by ckiosidis — APPROVED** [review]

File: suppressions.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0"?>
 <!--
   Copyright 2020 Spotify AB.
```
Comment (lupasarin) [inline suppressions.xml:3]: Shouldn't this line be removed here too?

File: spotbugs-exclude.xml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0"?>
 <!--
   Copyright 2020 Spotify AB.
```
Comment (lupasarin) [inline spotbugs-exclude.xml:3]: Shouldn't this line be removed here too?

**Review by lupasarin — APPROVED** [review]

File: suppressions.xml (hunk 1 above)
Comment (narape) [reply suppressions.xml:3]: Right, doing a find in the whole project showed more places. I will check further

**Review by narape — APPROVED** [review]

**Pull request merged**